shopping-cart/app/shoppingcart/checkout/confirmation/CheckoutConfirmationController.java

```diff
@@ -36,10 +35,9 @@ public CheckoutConfirmationController(final ControllerDependency controllerDepen
     public F.Promise<Result> show(final String languageTag) {
         final UserContext userContext = userContext(languageTag);
         final F.Promise<Cart> cartPromise = getOrCreateCart(userContext, session());
-        final Http.Context ctx = ctx();
         return cartPromise.map(cart -> {
             final CheckoutConfirmationPageContent content = new CheckoutConfirmationPageContent(cart, userContext, productDataConfig, i18nResolver(), reverseRouter());
-            final SunrisePageData pageData = pageData(userContext, content, ctx);
+            final SunrisePageData pageData = pageData(userContext, content, ctx());
```

Comment:
I'm actually modifying everything to follow that guideline. I think I will push the changes and dedicate a PR to follow some common standard everywhere.